addons/mail/models/discuss/mail_thread.py

```diff
@@ -0,0 +1,31 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from markupsafe import Markup
+
+from odoo import models, _
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _notify_thread(self, message, msg_vals=False, **kwargs):
+        rdata = super()._notify_thread(message, msg_vals, **kwargs)
+        self._notify_mentioned_channel(message, **kwargs)
```

Comment:
Would propagate msg_vals like in other ``_notify`` method (this parameter is probably going to disappear some day but hey, let us be coherent until we cleanup that part of the code :) )